### PR TITLE
user data saved on server on network access

### PR DESCRIPTION
**Pull request by st3inum**
try dilam.
login er somoy online theke data retrive korte hobe kintu

### CODE REVIEW

**Comment by st3inum** [comment]
jahhh

**Comment by Abdullahfoysal** [comment]
ki korli sagol

File: mathOlympiadV13/MathView-master/app/src/main/java/com/nishant/mathsample/signUpActivity.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+               long rowId= myDatabaseHelper.insertData(NAME,USERNAME,PASSWORD,GENDER,BIRTHDATE,EMAIL,PHONENUMBER,INSTITUTION,"0","0",DbContract.SYNC_STATUS_FAILED);
+               if(rowId==-1)
+               Toast.makeText(getApplicationContext(),"DATA Not SAVED TO LOCAL",Toast.LENGTH_SHORT).show();
+               else  Toast.makeText(getApplicationContext(),"DATA SAVED TO LOCAL",Toast.LENGTH_SHORT).show();
+            }
+        }
+
+    }
+    private void getAllFieldText(){
```
Comment (Abdullahfoysal) [inline mathOlympiadV13/MathView-master/app/src/main/java/com/nishant/mathsample/signUpActivity.java:65]: #
Duplicate of #

**Review by Abdullahfoysal — COMMENTED** [review]
nai

**Pull request merged**